doc/changelog.rst

```diff
@@ -9,6 +9,12 @@ Unrealeased
   When releasing next version, the "Unreleased" header will be replaced
   with appropriate version header and this help text will be removed.
 
+General
+~~~~~~~
+
+- Changed the way static resources are built.
+  Parcel is used to build all apps and `generated_resources.txt` is auto-generated.
```

Comment:
What's the point of generated_resources.txt if it's autogenerated?

It was supposed to list all of those files which should be excluded when checking if the resource generation cache is up-to-date or not. (I.e. currently the build system generates a hash for the state of the *input* files of the built application by hashing all files recursively, but excluding the files from `generated_resources.txt` (since those are output files not input files).) I can't see how this could work if generated_resources.txt is not there yet.

If you just want to get rid of generated_resources.txt, much better solution would be to improve the resource generation caching mechanisms to be smarter that it would know which files are input and which are output by some other means.

Reply:
Cool, I will think in a better solution for this